src/index.js

```diff
@@ -1,12 +1,47 @@
 import './css/base.scss';
-
 import User from './User.js';
 import Room from './Room.js';
 import Booking from './Booking.js';
+import Hotel from './Hotel.js';
 import domUpdates from './DOMUpdates.js';
 import dataFetcher from './dataFetcher.js';
 import loginHandler from './loginHandler.js';
 
 document.addEventListener('click', (event) => {
-  domUpdates.clickHandler(event);
+  clickHandler(event);
 })
+
+const clickHandler = (event) => {
+  if (event.target.id === 'log-in-btn') {
+    logIn();
+  } else if (event.target.id === 'username' || event.target.id === 'password') {
+    domUpdates.hideError();
+  }
+}
+
+const logIn = async () => {
+  const loginCredentials = domUpdates.checkLoginResponse();
+  if (loginCredentials.isValid) {
+    loginCredentials.username.includes('customer') ? await startCustomerApp(loginCredentials.username) : await startManagerApp();
+  } else {
+    domUpdates.displayError();
+  }
+}
+
+const startCustomerApp = async (username) => {
+  const customerID = loginHandler.validateCustomerID(username).customerID;
+  const currentCustomer = await dataFetcher.retrieveCustomerByID(customerID);
```

Comment:
I'm curious if it's going to matter later that currentCustomer is only stored in this function. In most cases it shouldn't if you'll be consistently updating data, but how, for instance, will you pass the customer's ID into a new booking?

Reply:
Really good point - I'll need to figure out how to let it persist. Might be an extension of how my login is happening right now